package.json

```diff
@@ -94,12 +94,12 @@
     "html-webpack-plugin": "^3.2.0",
     "image-webpack-loader": "^4.6.0",
     "mini-css-extract-plugin": "^0.5.0",
-    "node-sass": "^4.11.0",
-    "optimize-css-assets-webpack-plugin": "^5.0.1",
+    "node-sass": "^4.12.0",
+    "optimize-css-assets-webpack-plugin": "^5.0.3",
     "prop-types": "15.6.2",
-    "react": "^16.8.6",
+    "react": "^16.9.0",
     "react-a11y": "1.1.0",
-    "react-dom": "^16.8.6",
+    "react-dom": "^16.9.0",
     "react-redux": "5.1.1",
     "react-router": "3.0.5",
```

Comment:
Yeah, it's still fine.  It's that "componentWillMount is deprecated" thing - it will still work until react 17.  I think I'll just take a task to re-write all the deprecated methods sometime soon!